### ISSUE TITLE
Is callback necessary?

### ISSUE THREAD

**Issue by mrdulin** [scarney81/pg-hstore#23]
I read the source code about `hstore.stringify`: https://github.com/scarney81/pg-hstore/blob/master/lib/index.js#L32

```js
if (!callback || callback === null) return joined;
callback(joined);
```

It also works fine without passing `callback`  argument. Like this:

```js
const hstore = require('pg-hstore')();
const dataParsed = hstore.parse(data);
const dataStringified = hstore.stringify(data);
```

Am I correct?

**Comment by scarney81** [comment]
You are correct. This callback is not necessary. If I were re-writing this package I would not have constructed it with a callback signature.

**Comment by mrdulin** [comment]
@scarney81 Thanks for reply.  The documentation should be updated.

**Comment by SayakMukhopadhyay** [comment]
Its more than a year but would you like me to make an update to the README? @scarney81

**Comment by scarney81** [comment]
That would be most welcome @SayakMukhopadhyay
